mcaleb808/myPharmBackend
# Lock the Dependencies to Specific Versions

---

#### Description of changes made

> A described in #12, the dependencies should be locked to specific versions to avoid backward incompatibility crash issues.

#### Type of change

> **Please select the relevant options**


#### How Has This Been Tested?

> **Please describe the tests that you ran to verify your changes**


#### How should this be manually tested?

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](https://github.com/mcaleb808/myPharmBackend.git), and `cd into the project`
- Check out this branch by `git checkout lock-the-project-dependencies`
- Run `yarn | npm install` to install the project dependencies
- Make sure to have your `.env` file set up correctly
- Run `yarn dev | npm run dev` to start the local server.

#### Checklist:


#### GitHub issue your changes fix 

Resolves #12 

#### Screenshots

-None
